criu/include/sk-inet.h

```diff
@@ -69,14 +69,15 @@ extern int inet_connect(int sk, struct inet_sk_info *);
 
 #ifdef CR_NOGLIBC
 #define setsockopt sys_setsockopt
+#define pr_pwarn(fmt, ...) pr_warn(fmt ": %d\n", ##__VA_ARGS__, ret)
 #endif
 static inline void tcp_repair_off(int fd)
 {
 	int aux = 0, ret;
 
 	ret = setsockopt(fd, SOL_TCP, TCP_REPAIR, &aux, sizeof(aux));
 	if (ret < 0)
-		pr_err("Failed to turn off repair mode on socket\n");
+		pr_pwarn("Failed to turn off repair mode on socket %d", fd);
```

Comment:
It is the error. It is ignored only because we can't stop the restore at this point.

Reply:
Leaving at error level then.